iocore/net/SSLUtils.cc

```diff
@@ -1583,7 +1583,12 @@ SSLInitServerContext(const SSLConfigParams *params, const ssl_user_config &sslMu
     SSL_CTX_set_default_passwd_cb_userdata(ctx, &ud);
   }
 
-  if (sslMultCertSettings.cert) {
+  if (!sslMultCertSettings.cert) {
+    if (sslMultCertSettings.opt != SSLCertContext::OPT_TUNNEL) {
```

Comment:
This breaks the `TSSslServerContextCreate` plugin API method.

That method calls `SSLCreateServerContext` that delegates to `SSLInitServerContext` with an empty `sslMultCertSettings`.

This is useful for plugins that want to set the certificate/ca chain/private key themselves, for example to allow lazy loading of SNI certificates...